### ISSUE TITLE
🐛 Bug Report: dist directory should probably be gitignored

### ISSUE THREAD

**Issue by ClaudiuCeia** [appwrite/appwrite#3347]
### 👟 Reproduction steps

Not as much a bug but it felt like the best category. The `dist` directory in `public` is basically a compiled copy of the JS and CSS assets, and it feels like something that should be generated during deployment rather than part of the versioned repository. 

In the same vein, although I haven't yet confirmed this, `development` setup should run the uncompiled sources for easier debugging, no? 

### 👍 Expected behavior

In the `development` environment, we should execute the uncompiled/unminified/non-bundled JS and CSS sources which makes for easier debugging. 

In `production` (or `staging`, what have you) building the artifacts should be part of the deployment process (CI/CD pipeline, Github Actions, what have you)

### 👎 Actual Behavior

Making CSS or JS changes requires modifying the source code, running the gulp build pipeline and restarting the docker container so it reloads the just built artifacts. This is a bit of a pain in a world where hot reloading is so common. There's also no good system in place to ensure that the built artifacts (`dist` folder) is actually in sync with the JS and CSS source files as far as I can tell. 

### 🎲 Appwrite version

Version 0.14.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by TorstenDittmann** [comment]
Yes, we are aware of that issue. Problem is we are using the dist content and a bit more for our homepage to share css right now, which is included using composer.

Composer will include the raw git files and therefore we need dist right now.

Removing this is already being worked on behind the scenes 👍
